cpu/esp32/Kconfig

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2020 Gunar Schorcht
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+menuconfig KCONFIG_CPU_ESP32
+    bool "Configure ESP32 CPU"
+    depends on MODULE_ESP_ESP32
+    help
+        Configure ESP32 CPU via Kconfig.
+
+if KCONFIG_CPU_ESP32
+
+choice ESP32_DEFAULT_CPU_FREQ_MHZ
```

Comment:
There is no need to associate a symbol to the choice here, as it is not extended somewhere else.